dat/scripts/numstring.lua

```diff
@@ -23,8 +23,7 @@ end
 --    @return A string taking the form of "X ¤".
 --]]
 function creditstring( credits )
-   return gettext.ngettext( "%s ¤", "%s ¤", credits ):format(
-         numstring(credits) )
+   return _("%s ¤"):format( numstring(credits) )
```

Comment:
Is this desirable? My intuition was that using ngettext could theoretically be useful and doesn't hurt anything, though I could be wrong.

🕵️

Reply:
All right, I can back that bit out.
Sometime soon, I think we should get this and credits2str doing the same thing, so this may look different soon either way.
(Otherwise we get weird effects like "Pirate: Yar, pay me 23,456 ¤. .... Pay 23456 ¤?" when both functions are styling part of a message.)

Reply:
They don't need to do the same thing, but credits2str's output should be made translatable.

🕵️